### ISSUE TITLE
Sidebar layout change breaks Duo-Strength

### ISSUE THREAD

**Issue by Ban3** [ToranSharma/Duo-Strength#212]
This error gets logged to the console:

```
Uncaught (in promise) DOMException: Failed to execute 'insertBefore' on 'Node': The node before which the new node is to be inserted is not a child of this node.
    at displayTotalStrenthBox (chrome-extension://oghbejipobmlmgfbdjmfgnnhepngcmle/duoStrength.js:3626:17)
    at addFeatures (chrome-extension://oghbejipobmlmgfbdjmfgnnhepngcmle/duoStrength.js:4531:13)
```

This is caused by the actual sidebar being nested inside an additional div. Changing the sidebar selector to account the change restores the extension:

``` diff
-const SIDEBAR = "_1YfQ8";
+const SIDEBAR = "_1YfQ8 div:first-of-type";
```

But the layout change seems somewhat unintentional and might be temporary.

**Comment by ToranSharma** [comment]
@Ban3,

Thanks for reporting. This should be fixed by a9ead7bb4ac31251fc261ddf8b8d60d40779d280.
I will release this shortly as v2.0.26.

Let me know if you have any issues with anything after the update, or it if changes back.

Thanks,
Toran

**Comment by Ban3** [comment]
There's a leftover reference to `SIDEBAR` that is now undefined:
https://github.com/ToranSharma/Duo-Strength/blob/0b6c7b643331bd5ca8ce138e224c1cae2e8e790a/duoStrength.js#L4125

```
Uncaught (in promise) ReferenceError: SIDEBAR is not defined
    at arrangeSidebarBoxOrder (duoStrength.js:4125:67)
    at addFeatures (duoStrength.js:4553:9)
```

**Comment by ToranSharma** [comment]
Whoops pushed the update put fairly quickly last night, thanks for letting me know.

I'll try to fix this as soon as I can.

**Comment by ToranSharma** [comment]
This should be fixed in v2.0.27(#214) which I have just published

Thanks again,

Toran
